<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contrôle et supervision Lampadaire</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pbConnexion">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Connecter</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Contrôle</string>
    </property>
    <widget class="QPushButton" name="pbPWM">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Publier</string>
     </property>
    </widget>
    <widget class="QSlider" name="slidePWM">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>221</width>
       <height>19</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>5</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>LED 13</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbSlider">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Supervision</string>
    </property>
    <widget class="QLabel" name="lbHum">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>291</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressPWM">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>261</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Etat de la LED</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>slidePWM</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lbSlider</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
